src/main/webapp/WEB-INF/web.xml

```diff
@@ -93,6 +93,6 @@
 
   <servlet-mapping>
     <servlet-name>ProfileServlet</servlet-name>
-    <url-pattern>/profile</url-pattern>
+    <url-pattern>/profile/*</url-pattern>
   </servlet-mapping>
```

Comment:
Not sure if the asterisk was intended, but wanted to bring it to your attention if it wasn't